docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/druva-source.md

```diff
@@ -0,0 +1,144 @@
+---
+id: druva-source
+title: Druva Source
+sidebar_label: Druva
+description: This document explains how to configure the Druva Cloud-to-Cloud source setup on the Sumo logic environment.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+The Druva source provides the ability to analyze and fetch event logs from the **Druva inSync** API and send it to Sumo Logic. The **Druva inSync** backs up endpoint data and cloud applications, such as Microsoft Office 365 and Salesforce. It also provides archiving, data compliance monitoring, legal hold management, monitoring and detection tools to discover ransomware and E-discovery.
+
+This integration accesses the Druva inSync API to retrieve audit events. API documents can be found
+[here](https://developer.druva.com/docs/event-apis).
+
+
+## Prerequisites
+
+Before you begin setting up Druva source, you must meet the following requirements:
+* An administrator account is required for **Druva inSync** Cloud.
+* Enable the **Export Events** option. To know the steps, see the [Configure Druva inSync](#configure-inSync-to-export-events) section.
+* Obtain access tokens by [authentication](#authentication).
+
+
+### Configure Druva inSync to export events
+
+Only an Druva inSync Cloud administrator can enable the option to export events and define the Events API settings. To get the event logs, follow the steps to enable the **Export Events**:
+1. Log in to the Druva inSync Management Console with [inSync Cloud](https://login.druva.com/) account or [inSync GovCloud](https://loginfederal.druva.com/) account.
+2. On the [inSync Management Console](https://docs.druva.com/Endpoints/030_Set_up_inSync_for_Endpoints/010_Initial_Configuration/010_Sign_in_to_inSync_Management_Console/Sign_in_to_inSync_Management_Console) menu bar, click **Settings** (wheel icon) > Settings. The Settings page appears.
+3. Click the **inSync APIs** tab.
+4. In the Events API settings area, click **Edit**. The Edit Events API Settings window appears.<br/><img src={useBaseUrl('img/send-data/events-api-settings.png')} alt="events-api-settings.png" width="450"/>
```

Comment:
Broken image. It looks like you accidentally put the images in the docs folder  (`/docs/send-data`). Please move them to `/static/img/send-data`.